README.md

```diff
@@ -1,3 +1,23 @@
+## Merith-TK `sshcode` Fork
+___
+This fork is specifically for getting `sshcode` to work within the `msys`/`MinGW` enviroment
+
+What currently works, 
+* pretty much everything you would expect if you dont have chrome installed, starts the reverse proxy server, opens default browser.
+
+What doesnt work
+* Chrome support, cannot test as i dont use chrome.
+* Specifying a file path, for some odd reason, its still using windows filepath when trying to access the server with something like `sshcode --skipsync merith@192.168.0.101:/opt` will try and access `C:/mingw/opt` which, of course, does not exist. assistance is highly welcome.
+
+What KINDA works (Requires user to do some stuff)
+* SyncBack??? <> Basically, you need rsync installed, just like on linux
+* VSCode SyncBack <> It will use the DEFAULT paths to store VS code information on windows
+### TODO
+[ ] Detect Windows seperate from `msys`/`mingw` (Probably going to check for unix variables.)
+[ ] Add windows support (IE: `C:\` file paths )
+
+
+___
```

Comment:
This whole section should be removed. If you want to track TODOs please do so in the PR description.

Reply:
I mostly track TODOs in Readme so I can easily find them should I find myself completely offline